suite-native/device-manager/src/components/WalletList.tsx

```diff
@@ -1,20 +1,59 @@
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
 
-import { selectDeviceInstances } from '@suite-common/wallet-core';
+import {
+    createDeviceInstanceThunk,
+    selectDevice,
+    selectDeviceInstances,
+    selectDeviceState,
+    selectIsPortfolioTrackerDevice,
+} from '@suite-common/wallet-core';
 import { VStack } from '@suite-native/atoms';
 import { TrezorDevice } from '@suite-common/suite-types';
 
 import { WalletItem } from './WalletItem';
+import { selectHasNonPassphraseDevice } from '@suite-native/device/src/selectors';
 
 type WalletListProps = {
     onSelectDevice: (device: TrezorDevice) => void;
 };
 
 export const WalletList = ({ onSelectDevice }: WalletListProps) => {
+    const dispatch = useDispatch();
     const devices = useSelector(selectDeviceInstances);
+    const selectedDevice = useSelector(selectDevice);
+    const hasNonPassphraseDevice = useSelector(selectHasNonPassphraseDevice);
+    const deviceState = useSelector(selectDeviceState);
+    const isPortfolioTrackerDevice = useSelector(selectIsPortfolioTrackerDevice);
+
+    const isSelectable = devices.length > 1 || !hasNonPassphraseDevice;
+
+    // we want to show placeholder in case there is only passphrase wallet without standard and not portfolio
+    const showPlaceholder = !hasNonPassphraseDevice && deviceState && !isPortfolioTrackerDevice;
+
+    // on tap of placeholder we actually create device with empty passphrase and select it
+    const handleOnPlaceholderPress = async () => {
```

Comment:
nit: There is an unwritten rule that the component prop functions starts with prefix `on...` and the functions that handle their behaviour (are passed there) are named `handle...` [More here.](https://medium.com/@master_43681/understanding-and-proper-usage-of-on-and-handle-prefixes-in-react-function-naming-119f0beea400)
```suggestion
    const handlePlaceholderPress = async () => {
```

Reply:
bc094e834e